src/components/tasks/SuggestionBox/SuggestionBox.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import SuggestionList from './SuggestionList';
+import Loading from './Loading';
+import UserNotFound from './UserNotFound';
+
+type Props = {
+    onClickName: (userName: string) => void;
+    loading: boolean;
+    suggestions: GithubInfo[];
+};
```

Comment:
can we please export these types from .type file

Reply:
I have commented down below regarding this @Pratiyushkumar